packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -137,7 +132,6 @@ const TextareaField = (props) => {
     required,
     showOptional,
     onChange,
-    placeholder,
```

Comment:
Same as this comment: https://github.com/cerner/terra-core/pull/3075#discussion_r457557232

Reply:
Update here https://github.com/cerner/terra-core/pull/3075/commits/ad81efd1977642f263c01c46fda0a8f65f4ea6ba